README.md

```diff
@@ -7,6 +7,9 @@ Compatible with the [`graphql-batch` gem](https://github.com/Shopify/graphql-bat
 
 Be sure to read the [CHANGELOG](CHANGELOG.md) to stay updated on feature additions, breaking changes made to this gem.
 
+**NOTE**: Not tested with graphql-ruby's multiplexing feature. Metrics may not
```

Comment:
@chrisbutcher is this right? I see some tests for multiplexes that look correct, but I also see some code in the tracer which just seems... blatantly unaware of multiplexing and would override previous values for each new query in the multiplex?

Reply:
The statement appears outdated.

As far as the tracer's multiplex awareness, that should not be the case. The query context it stores values in is specific to each query in a multiplex. I can dig in/pair on this if it's still confusing.

Reply:
🤔 AFAICT the tracer's `validate` event only occurs once even for multiplexed queries, but it receives a context (which one?) and that's the context we store our lex/parse times in... How does this work?

Reply:
> The README statement appears outdated.

I didn't realize yesterday that the line this thread is spun from is one that you just added. That was probably fairly confusing too 🙃 .

More importantly, 🤦  I can confirm that both lex and parse suffer from a "last write wins" issue here, since `pre_context` is not multiplex-aware.

`validate` does fire more than once, and does correctly store its own timings into each query's own context within a multiplex, but it also uses the singular, "last write wins" values for lex and parse timings.

I'm prepping a small write-up to highlight what is broken here and why, with a step-by-step example.

Reply:
See https://github.com/Shopify/graphql-metrics/issues/45